RobotCode/snobot2018/src/org/snobot/leds/DostarLedStrip.java

```diff
@@ -10,11 +11,14 @@
     private static final int sSPI_CLK_RATE = 13000000;
     private static final int sMAX_BYTES_PER_MESSAGE = 124; // Max size the hardware supports is 127.  Rounding down to an 4 byte boundary
 
+    private static final double sMAX_BRIGHTNESS = 1; // Brightness scaling factor, 0-1
```

Comment:
We can tweak this to make them not "SUPERBRIGHTLEDS.com" and make them just "PRETTY BRIGHT LEDs".  We also theoretically can draw too much current if we turn them all on white at the same time, so this would be a quick fix